torch/distributed/nn/functional.py

```diff
@@ -79,6 +79,28 @@ def reduce(tensor, dst, op=dist.ReduceOp.SUM, group=dist.group.WORLD):
     return _Reduce.apply(dst, op, group, tensor)
 
 
+def reduce_scatter(output, input_tensors, op=dist.ReduceOp.SUM, group=dist.group.WORLD):
```

Comment:
Lets have 1:1 parity with this API: https://github.com/pytorch/pytorch/blob/master/torch/distributed/distributed_c10d.py#L2408 (input_list is the parameter name and not input_tensors)

Reply:
sure.